module/VuFind/src/VuFind/Db/Table/Tags.php

```diff
@@ -516,26 +518,6 @@ public function getTagList($sort, $limit = 100, $extra_where = null)
         return $tagList;
     }
 
-    /**
-     * Delete a group of tags.
-     *
-     * @param array $ids IDs of tags to delete.
-     *
-     * @return void
-     */
-    public function deleteByIdArray($ids)
```

Comment:
Did you mean to delete this? I think it's still currently being used by VuFind\Db\Table\ResourceTags. Is there something else you meant to delete instead, maybe from the Doctrine code rather than the Laminas code?

Reply:
It was migrated to TagService and no longer useful. It was removed intentionally.

Reply:
Ahh, sorry, I got confused because ResourceTags::processDestroyLinks hasn't been deleted yet. But of course that's what we just discussed -- I will work on deleting that and other obsolete methods as part of my testing process. :-)